### DISCUSSION TITLE
Searching domain names results in non-secure and fails (DNS Issue)

### DISCUSSION THREAD

**Discussion by noosemanAL** [searxng/searx-instances#554] [Report bug public instance] [answered]
Searching domains like privacy.com searches with http which fails, How can I enable https searches?
![privacy](https://github.com/searxng/searx-instances/assets/52615094/60c40b58-512d-4943-86a5-0cb78794e03a)
used search.disroot.org and search.lollipopstealer.com

**Comment by return42** [comment, marked as answer]
By default SearXNG returns https only ..

> used search.disroot.org and search.lollipopstealer.com

disroot.org is heavily customized, customized instances tend not to be updated regularly / disroot.org  is not rebased since a longer time (2024.3.11+542f7d0d7) / search.lollipopstealer.com does not exists.

Use one of the updated SearXNG instances on https://searx.space
